src/validation/transaction.cairo

```diff
@@ -176,13 +228,15 @@ mod tests {
                 }
             ]
                 .span(),
-            lock_time: 0
+            lock_time: 500001
```

Comment:
Please do not edit tests for other validations, create a new one. Unit test has to test only `validate_relative_locktime` function, not the `validate_transaction`.

New tests (create one for each scenario) should cover scenarios:
- relative locktime is disabled
- relative locktime is enabled/block height
- relative locktime is enabled/block time

There are examples in https://learnmeabitcoin.com/technical/transaction/input/sequence/ that you can use.